addons/l10n_in_edi_ewaybill/views/account_move_views.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="invoice_form_inherit_l10n_in_edi_ewaybill" model="ir.ui.view">
+        <field name="name">account.move.form.inherit.l10n.in.ewaybill</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook/page[@name='other_info']" position="before">
+                <page string="eWayBill" name="l10n_in_edi_ewaybill_page"
+                    attrs="{'invisible':[('move_type', '=', 'entry'), ('country_code', '!=', 'IN')]}">
+                    <field name="l10n_in_edi_ewaybill_direct_api" invisible="1"/>
+                    <group name="ewaybill_group">
+                        <group string="Transaction Details" name="Transaction_group" 
+                            attrs="{'invisible': [('l10n_in_edi_ewaybill_direct_api', '!=', True)]}">
+                            <field name="l10n_in_transaction_type" string="Transaction Type"/>
+                            <field name="l10n_in_type_id"
+                                attrs="{'invisible': [
+                                    ('move_type','in',('in_invoice','in_refund','in_receipt'))]}"
+                                domain="[
+                                    ('allowed_in_supply_type','in',('out','both')),
+                                    ('parent_type_ids', '=', False)]"/>
+                            <field name="l10n_in_subtype_id"
```

Comment:
is there no way to combine type and subtype (it seems like many fields for the same thing and it is not because it is 3 tags in the XML that we need 3 fields)

Reply:
see this mapping and as per this https://docs.ewaybillgst.gov.in/apidocs/sub-docType-mapping.html
we need to pass type and subtype in JSON

Reply:
if you have any ideas to combine it and still we pass it separately in JSON then tell me I apply it